package.json

```diff
@@ -82,16 +82,18 @@
   ],
   "jspm": {
     "main": "js/bootstrap",
+    "ignore": ["dist/js/npm"],
+    "directories": {
+      "lib": "dist"
+    },
     "shim": {
       "js/bootstrap": {
-        "deps": "jquery",
+        "deps": ["jquery"],
         "exports": "$"
       }
     },
-    "files": [
-      "css",
-      "fonts",
-      "js"
-    ]
+    "dependencies": {
+      "jquery": "github:components/jquery"
```

Comment:
Sure, there might be benefits to being able to resolve version ranges in the browser, which is made much easier by sticking to simple ranges (can replace node-semver with the much-smaller https://github.com/systemjs/systemjs/blob/0.16.11/lib/extension-versions.js). Still seeing how it works out, but so far haven't found it to be too restricting that this limitation has been critical yet.